https://patch-diff.githubusercontent.com/raw/xmms2/xmms2-devel/pull/7.patch


From 6b45a72568b8cf6db6103d386dd371f6ce285785 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Sat, 2 Oct 2021 21:30:18 +0100
Subject: [PATCH] wscript: default to -O2, not -O0

Let's make './waf configure' mostly usable for distribution packaging
without CFLAGS/CXXFLAGS tweaks.
---
 wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript b/wscript
index 44871c687..b7b688ddf 100644
--- a/wscript
+++ b/wscript
@@ -377,7 +377,7 @@ def configure(conf):
                 hasoptim=True
                 break
         if not hasoptim:
-            conf.env.append_unique(env, ['-g', '-O0'])
+            conf.env.append_unique(env, ['-g', '-O2'])
 
     if conf.options.enable_gcov:
         conf.env.enable_gcov = True
